src/renderer/components/chat/ChatList.tsx

```diff
@@ -68,18 +74,32 @@ export default function ChatList(props: {
     selectChat(chatId)
   }
 
+  const isSearchActive = queryStr !== ''
+
   const renderAddContactIfNeeded = () => {
-    if (queryStr === '' || chatListIds.length > 0) return null
+    if (chatListIds.length > 0) return null
```

Comment:
@Simon-Laux  did you miss a isSearchActive here?

Reply:
nope it is checked bellow (L116), no reason to check it twice.